src/main/resources/edu/cornell/kfs/cu-spring-kfs.xml

```diff
@@ -3,27 +3,28 @@
 	   xmlns:p="http://www.springframework.org/schema/p"
 	   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
        xsi:schemaLocation="http://www.springframework.org/schema/beans http://www.springframework.org/schema/beans/spring-beans-2.0.xsd">
-	
-	<!-- Import KEW RouteHeaderService, which is not auto-GRL-loaded by the base KFS beans. -->
-	<bean id="enDocumentRouteHeaderService" p:serviceName="enDocumentRouteHeaderService" parent="grlBeanImporter" />
-	<!-- Import KRAD CriteriaLookupService, which is not auto-GRL-loaded by the base KFS beans. -->
-	<bean id="criteriaLookupService" p:serviceName="criteriaLookupService" parent="grlBeanImporter" />
-	
-	<import resource="sys/cu-spring-sys.xml" />
-	<import resource="coa/cu-spring-coa.xml" />
-	<import resource="module/cg/cu-spring-cg.xml" />
-	<import resource="gl/cu-spring-gl.xml" />
-	<import resource="fp/cu-spring-fp.xml" />
-	<import resource="sec/cu-spring-sec.xml"/>
-	<import resource="vnd/cu-spring-vnd.xml" />
-	<import resource="module/purap/cu-spring-purap.xml" />
-	<import resource="module/ld/cu-spring-ld.xml" />
-	 
-	<import resource="pdp/cu-spring-pdp.xml" />
-	<import resource="module/cam/cu-spring-cam.xml" />
-	<import resource="module/ezra/cu-spring-ezra.xml" />
-	<import resource="module/receiptProcessing/cu-spring-receiptProcessing.xml" />
-	
-	<import resource="tax/cu-spring-tax.xml" />
-	<import resource="concur/cu-spring-concur.xml" />
+
+    <!-- Import KEW RouteHeaderService, which is not auto-GRL-loaded by the base KFS beans. -->
+    <bean id="enDocumentRouteHeaderService" p:serviceName="enDocumentRouteHeaderService" parent="grlBeanImporter" />
+    <!-- Import KRAD CriteriaLookupService, which is not auto-GRL-loaded by the base KFS beans. -->
+    <bean id="criteriaLookupService" p:serviceName="criteriaLookupService" parent="grlBeanImporter" />
+    
+    <import resource="sys/cu-spring-sys.xml" />
```

Comment:
I think it would have been easier to spot what changed in this file if only the alignment for the one changed line was fixed.

Reply:
I agree however, the alignment would have been off as our recommended standard indent is 4 spaces, but I think a lot of the files are saved with the tab character or 2/6 spaces for each indent. I believe this is the cause of the formatting problem as it looks fine in eclipse